client/dev/definitions/abilities.jsx

```diff
@@ -0,0 +1,227 @@
+const AbilityDefs = [
+    {
+      name: "Strength",
+      description: "Determines how much damage you can deal in melee and brawling situations, plus how much you can lift and carry.",
+      id: 1,
+      modifiers: {
+        fractional: true,  // for later code to do an if else.  Fractional = true means your modifier depends on your Ability Score + whether your Fractional Ability Score > 50 
+        statsets: [ //Modifiers are responsible for modifying more than one thing hence the array 
+          {
+            name: "Damage",
+            feature: true, //Search for the attribute table's feature modifier by finding the one which is marked feature: true 
+            values: [    // Because the modifier is fractional we have an array of arrays: [A, B] where A is if your Fractional is < 50 and B is if > 50. 
```

Comment:
Might be better to provide a name of a function to determine the values for a given statset